WcaOnRails/app/javascript/auto-numeric/index.js

```diff
@@ -0,0 +1,67 @@
+import AutoNumeric from 'autonumeric';
+import currenciesData from 'wca/currenciesData.js.erb';
+
+let autoNumericObjectMap = new Map();
+
+function getCurrencyInfo(isoCode) {
+  return currenciesData.byIso[isoCode] || currenciesData.byIso.USD;
+}
+
+// Create a mask for an amount input in the given currency, using autoNumeric
+// 'action' can be either "init" or "update"
+// 'element' is a jquery element for the targeted input field
```

Comment:
nit: our convention is to prefix variable names with `$` if they refer to jquery objects, so could you call this `$element` instead?